sql/src/main/antlr/OpenDistroSQLParser.g4

```diff
@@ -166,16 +166,16 @@ scalarFunctionName
     ;
 
 mathematicalFunctionName
-    : ABS | CEIL | CEILING | CONV | CRC32 | EXP | FLOOR | LN | LOG | LOG10 | LOG2 | MOD | POW | POWER
-    | ROUND | SIGN | SQRT | TRUNCATE
+    : ABS | CEIL | CEILING | CONV | CRC32 | E | EXP | FLOOR | LN | LOG | LOG10 | LOG2 | MOD | PI | POW | POWER
+    | RAND | ROUND | SIGN | SQRT | TRUNCATE
     ;
 
 dateTimeFunctionName
     : DAYOFMONTH
     ;
 
 functionArgs
-    : functionArg (COMMA functionArg)*
+    : (functionArg (COMMA functionArg)*)?
```

Comment:
is this not necessary because of `functionArgs?` in `functionCall`? And I think it's more appropriate to mark this optional in `functionCall` for PPL too?

Reply:
Yes functions with no argument will return an empty collection as `functionArgs` in parser. It has also been refracted in PPL.

Reply:
I was asking is this change necessary because:

```
functionCall
    : scalarFunctionName LR_BRACKET functionArgs? RR_BRACKET        #scalarFunctionCall
    ;
```